geodesic_path/mesh.h

```diff
@@ -31,8 +34,8 @@ class Mesh {
 };
 
 inline double Area(const Triangle &triangle) {
-  const Point a_b{ToVector(triangle.a, triangle.b)};
-  const Point a_c{ToVector(triangle.a, triangle.c)};
+  const Point a_b{ToVector(vertices_[triangle.a], vertices_[triangle.b])};
+  const Point a_c{ToVector(vertices_[triangle.a], vertices_[triangle.c])};
```

Comment:
vertices_ is private in mesh, so the Area function doesn't see it.

Reply:
Good point, it could be refactored to a private member?